README.md

```diff
@@ -233,6 +233,7 @@ This section lists each configuration option, and whether it can be set by each
 | Kubernetes Service annotation to set EIP metro                                                                                                               |                | `METAL_ANNOTATION_EIP_METRO`            | `annotationEIPMetro`           | `"metal.equinix.com/eip-metro"`                              |
 | Kubernetes Service annotation to set EIP facility                                                                                                            |                | `METAL_ANNOTATION_EIP_FACILITY`         | `annotationEIPFacility`        | `"metal.equinix.com/eip-facility"`                           |
 | Tag for control plane Elastic IP                                                                                                                             |                | `METAL_EIP_TAG`                         | `eipTag`                       | No control plane Elastic IP                                  |
+| ID for control plane Equinix Metal Load Balancer                                                                                                                             |                | `METAL_LOAD_BALANCER_ID`                         | `loadBalancerID`                       | No control plane Equinix Metal Load Balancer                                  |
```

Comment:
I think this option would fit with how we offer arguments in `METAL_LOAD_BALANCER`, as used by [metallb](https://github.com/kubernetes-sigs/cloud-provider-equinix-metal/blob/45dde1a565767e17e4a515a092a46c5ee4f3e744/metal/loadbalancers.go#L81-L89):
* `METAL_LOAD_BALANCER=emlb://sv`
* `METAL_LOAD_BALANCER=emlb://?loadbalancerID=123`
* `METAL_LOAD_BALANCER=emlb://?tags=cpem`

Reply:
I know there is no support for tags today.

I'm curious how well CPEM will handle the limitation of working with a single LoadBalancer and whether this is more of a development feature or if this is an argument that users will want in their clusters.

Reply:
Alternatively, if new options are the preferred approach, perhaps with #472, we may want to expose arguments in the future for the metallb options only offered through `METAL_LOAD_BALANCER` query parameters today.